src/server/command_handler.py

```diff
@@ -17,17 +17,44 @@
 
 from src.utils.logging_manager import LoggingManager
 
+from src.parser.parser import Parser
+from src.optimizer.statement_to_opr_convertor import StatementToPlanConvertor
+
+from src.executor.plan_executor import PlanExecutor
+from src.optimizer.plan_generator import PlanGenerator
+from src.utils.logging_manager import LoggingManager
+
 
 @asyncio.coroutine
-def handle_request(transport, request_message):
+def handle_request(transport, query):
     """
         Reads a request from a client and processes it
 
         If user inputs 'quit' stops the event loop
         otherwise just echoes user input
     """
+    parser = Parser()
+    eva_statement = parser.parse(query)
+    insert_statement = eva_statement[0]
+
+    LoggingManager().log("Result from the parser: " + str(insert_statement))
+
+    convertor = StatementToPlanConvertor()
+    convertor.visit(insert_statement)
+
+    convertor.plan
+
+    eva_statement = parser.parse(query)
+    response_message = ""
+
+    for i in range(len(eva_statement)):
 
-    response_message = "foo"
+        LoggingManager().log("Parse Tree: " + str(eva_statement[i]))
+        logical_plan = StatementToPlanConvertor().visit(eva_statement[i])
+        physical_plan = PlanGenerator().build(logical_plan)
+        df = PlanExecutor(physical_plan).execute_plan()
+        statement_response_message = ','.join(map(str, df.collect()))
+        response_message += statement_response_message
```

Comment:
It would be good to represent response_message as list.
Where each entry in the list corresponds to a statement in eva_statements.

Reply:
It makes sense to keep response_messages as a list but I'm worried if asyncio message passing allows list. In case it doesn't we have to convert the list back to string.